### PR TITLE
add netdoctor docs

### PR DESCRIPTION
**Pull request by rxy0210**
**What type of PR is this?**

**What this PR does / why we need it**:
add netdoctor docs
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by rxy0210** [comment]
@duanmengkk help review plz

File: docs/tutorials/netdoctor_ZH.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+Do check... 100% [================================================================================]  [0s]
++-----+----------------+----------------+-----------+-----------+
+| S/N | SRC NODE NAME  | DST NODE NAME  | TARGET IP |  RESULT   |
++-----+----------------+----------------+-----------+-----------+
+|   1 | ecs-net-dr-002 | ecs-net-dr-001 | 10.0.1.86 | SUCCEEDED |
+|   2 | ecs-net-dr-001 | ecs-net-dr-002 | 10.0.2.29 | SUCCEEDED |
++-----+----------------+----------------+-----------+-----------+
+````
+
+* `netctl clean`命令用于清理`NetDoctor`创建的所有资源。
```
Comment (duanmengkk) [inline docs/tutorials/netdoctor_ZH.md]: 1. The location of the Chinese documentation is incorrect; please refer to the README.
2. Please supplement the documentation for testing single-cluster pod-to-pod, pod-to-node DNS service, as well as all functionalities for multi-cluster, writing from the user's perspective.

Comment (rxy0210) [reply docs/tutorials/netdoctor_ZH.md]: Submitted.
1. Changed the location of the Chinese document
2. Added four modes of mode (pod, node, node-to-pod, pod-to-node), added three modes of protocol (' tcp ', 'udp', 'dns'), added some parameter explanation and specific code explanation

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/tutorials/netdoctor.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+* Kubernetes集群投入使用后集群网络可能会存在种种的连通性问题，因此我们希望可以有一个验收工具，在完成部署后检查集群的网络连通性是否正常。
+
+* 另一方面，Kosmos是一个跨集群的解决方案，在Kosmos管理多个集群之前，需要先检查各个集群的容器网络自身是否存在问题，部署完成后也需要验证跨集群的网络是否已经被Kosmos连通。
+
+* 出于以上两个方面，我们设计了 [NetDoctor](https://github.com/kosmos-io/netdoctor) 工具用于解决Kubernetes集群遇到的网络问题。
+
+## 架构
+
+![NetDoctor](../getting-started/img/netdr-arch.png)
```
Comment (yuleichun-striving) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/tutorials/netdoctor.md]: The image needs to be imported into the module to which it belongs

**Pull request merged**